app/experimenter/experiments/migrations/0201_nimbusexperiment_firefox_max_version.py

```diff
@@ -0,0 +1,115 @@
+# Generated by Django 3.2.10 on 2021-12-14 20:36
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("experiments", "0200_nimbusexperiment_warn_feature_schema"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="nimbusexperiment",
+            name="firefox_max_version",
+            field=models.CharField(
+                choices=[
+                    ("", "No Version"),
+                    ("11.!", "Firefox 11"),
+                    ("12.!", "Firefox 12"),
```

Comment:
Was tempted to remove the choices here from both min & max version in the DB model to avoid migrations like this in the future when versions change. But, didn't want to rock the boat too much. If it's worth it, I can tweak things to drop the choices though

Reply:
No yeah good thinking leaving it alone.  I don't know how much value there really is in explicitly declaring choices right on models, it auto populates the admin but since we don't use djangos forms/templates in nimbus the value basically stops there.  Maybe in January we can do that as a nice cleanup step.